rero_ils/modules/cli.py

```diff
@@ -372,6 +384,9 @@ def create(infile, append, reindex, dbcommit, commit, verbose, debug, schema,
     click.echo('DB commit: {count}'.format(count=count))
     db.session.commit()
 
+    if save_errors:
+        error_file.write(']')
```

Comment:
missing a `error_file.close()` at the end of the method. Previously it was made by the `with` statement, now i think you need do it yourself.

Reply:
ok